src/systems/tracked-controls.js

```diff
@@ -7,6 +7,7 @@ var utils = require('../utils');
  * It maintains a list with the available tracked controllers
  */
 module.exports.System = registerSystem('tracked-controls', {
+  schema: {alwaysAllowIdPrefixes: {type: 'array'}},
```

Comment:
Is this a problem in both Chromium and Firefox, the WebVR API implementation, the spec all of the above? I think the gamepad API should always return a pose if it corresponds to a tracked device. Do you have a consistent way to reproduce the issue. I can open a bug on Firefox / Gecko